DESCRIPTION.md

```diff
@@ -1,4 +1,6 @@
-My App
+Description
```

Comment:
The template says "My App" so should this say "Lanswepper" instead of "Description"

Reply:
No needed